README.md

```diff
@@ -178,6 +181,45 @@ docker-compose up -d op-geth-replica op-node-replica op-blockscout-replica
 
 Open the replica explorer ([http://127.0.0.1:4002/](http://127.0.0.1:4002/)). L2 block is synchronized with about a 1-minute delay. Synchronization can be sped up by reducing the `--verifier.l1-confs` in the op-node-replica service.
 
+## Setup Message Relayer
+
+1. Go to message_relayer folder
+```shell
+cd message_relayer
+```
+2. Set up env 
+
+```shell
+cp .env.example .env
```

Comment:
Please set env variable in docker-compse.yml
like this
```
  message-relayer:
    environment:
      L1_RPC: 'http://...'
      L2_RPC: 'http://...'
```
https://github.com/oasysgames/oasys-private-opstack/blob/main/docker-compose.yml#L249-L271